src/components/AddAdvert/AddAdvertForm/AddAdvertForm.jsx

```diff
@@ -0,0 +1,226 @@
+    return (
+        <Section>
+            <Formik
+                initialValues={initialValues}
+                onSubmit={handleSubmit}
+            >
+                {({
+                    errors,
+                    touched,
+                    values,
+                    handleChange,
+                    handleBlur,
+                    isSubmitting,
+                }) => (
+                    
+                    <div>
+                        <FormTitle>Create advertisement</FormTitle>
+                        <Form>
+                            {/* // TITLE */}
+                            <div>
+                                <label htmlFor='title'>Title</label>
+                                <Field
+                                    id='title'
+                                    type='text'
+                                    name='title'
+                                    placeholder='Enter title for your course'
+                                    value={values.title}
+                                    onChange={handleChange}
+                                    onBlur={handleBlur}
+                                    border={touched.title && errors.title}
+                                />
+                                <ErrorMessage name='title' component='div' />                                
+                            </div>
+
+                            {/* // CATEGORY */}
+                            <div>
+                                <label htmlFor='category'>Category</label>
+                                <Field
+                                    id='category'
+                                    as='select'
+                                    name='category'
+                                    placeholder='Select match category'
+                                    value={values.category}
+                                    onChange={handleChange}
+                                    onBlur={handleBlur}
+                                    border={touched.category && errors.category}
+                                >
+                                    <option value='language-learning'>Language learning</option>
+                                    <option value='it'>IT Sphere</option>
+                                    <option value='science'>Science and education</option>
+                                    <option value='art'>Art and creativity</option>
+                                    <option value='sport'>Sports and fitness</option>
+                                    <option value='personality-development'>Personality development</option>
+                                    <option value='health'>Health and harmony</option>
+                                </Field>
+                                <ErrorMessage name='category' component='div' />                                
+                            </div>
+
+                            {/* // SUBCATEGORY */}
```

Comment:
The code is clear enough, this comment is redundant

Reply:
has removed